_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -958,7 +1107,19 @@ link:https://docs.aws.amazon.com/general/latest/gr/aws_tasks-that-require-root.h
 the use of root].  Fortunately, most of these activities are rare, so usage of the root account can be kept to
 a minimum.
 
-Next, continue with the <<configure_logging>> section.
+[[configure_storage]]
+=== Configure storage
+
+[[s3_buckets_deployment]]
+==== S3 Buckets
+To make sure your S3 buckets are compliant with the benchmark, all you have to do is use the
+link:https://github.com/gruntwork-io/terraform-aws-security/tree/master/modules/private-s3-bucket[`private-s3-bucket` module].
```

Comment:
```suggestion
To make sure your S3 buckets are compliant with the benchmark, use the
link:https://github.com/gruntwork-io/terraform-aws-security/tree/master/modules/private-s3-bucket[`private-s3-bucket` module] to create and manage all of your S3 buckets. This module blocks public access and enforces encryption by default. Note that all Gruntwork modules that create S3 buckets use this module under the hood.
```

Reply:
Oh, should we add a `terragrunt.hcl` example of using this module? Or better yet, the service in `aws-service-catalog` that wraps it?

Reply:
Fixed in https://github.com/gruntwork-io/gruntwork-io.github.io/pull/392/commits/5cd3f1c19af4b37e0f4ecd1980e2da0f1e7b2b00.